script/testing/reporting/parsers/oltpbench/summary_parser.py

```diff
@@ -1,95 +1,58 @@
-#!/usr/bin/python3
-
 import json
 from time import time
 
-from reporting.utils import get_value_by_pattern
-from reporting.constants import UNKNOWN_RESULT, LATENCY_ATTRIBUTE_MAPPING
+from ...constants import LATENCY_ATTRIBUTE_MAPPING, UNKNOWN_RESULT
+from ...utils import get_value_by_pattern
 
 
 def parse_summary_file(path):
     """
-    Read data from file OLTPBench summary file.
+    Read data from an OLTPBench-generated summary file.
+
+    Parameters
+    ----------
+    path : str
+        Path to an OLTPBench-generated summary file.
+
+    Returns
+    -------
+    metadata : dict
+        An object containing metadata information.
+    timestamp : int
+        The timestamp when the benchmark was created, in milliseconds.
+        TODO(WAN): wtf is this?
```

Comment:
I think I should have provided a mor detailed comment, my confusion was/is:

why is your timestamp being given a unit of milliseconds? milliseconds since UNIX epoch?
A timestamp refers to a specific point in time

maybe I need to go back to OLTPBench to figure out what's going on here

Reply:
It looks like it comes from:
``` java
Date now = new Date();
summaryMap.put("Current Timestamp (milliseconds)", now.getTime());
```
So, the number of milliseconds since January 1, 1970, 00:00:00 GTM.

Reply:
thanks! added